angular-portfolio/src/app/material/material.module.ts

```diff
@@ -0,0 +1,22 @@
+
+import { NgModule } from '@angular/core';
+import { MatButtonModule } from '@angular/material/button';
+import { MatCardModule } from '@angular/material/card';
+import { MatTabsModule } from '@angular/material/tabs';
+import {MatToolbarModule} from '@angular/material/toolbar';
+import {MatButtonToggleModule} from '@angular/material/button-toggle';
+
+const Material = [
+  MatButtonModule,
+  MatCardModule,
+  MatTabsModule,
+  MatToolbarModule,
+  MatButtonToggleModule
+];
+
+@NgModule({
+  imports: [ Material ],
```

Comment:
This should be either "imports: material" or "imports: [...material]". Right now this has an array within another array.